ansible_collections/arista/avd/plugins/action/yaml_templates_to_facts.py

```diff
@@ -177,7 +195,7 @@ def run(self, tmp=None, task_vars=None):
 
                 # If output_schema is set, perform inplace conversion of each returned dict according to output_schema
                 # to normalize the data to correct format before merging
-                if output_schema:
+                if output_avdschema:
```

Comment:
should we change the comment above? - I am not sure if this change fixes some unworking thing?

Reply:
I just check on a var that is set if either output_schema or output_schema_it is set.